.circleci/config.yml

```diff
@@ -0,0 +1,157 @@
+    name: Run tests
+    command: yarn test:ci --maxWorkers=10
+
+  - &unit_test_with_coverage
+    name: Run tests
+    command: yarn test:coverage:ci --maxWorkers=10
+
+version: 2
+jobs:
+  greenkeeper_lockfile:
+    <<: *node8Defaults
+    steps:
+      - checkout
+      - restore-cache: *restore_yarn_cache
+      - run:
+          name: Install greenkeeper-lockfile-update
+          command: yarn global add greenkeeper-lockfile@1 --prefix ~/.yarn
+      - run: *yarn_install
+      - save_cache: *save_yarn_cache
+      - run:
+          name: Update greenkeeper-lockfile
+          command: ~/.yarn/bin/greenkeeper-lockfile-update
+      - run:
+          name: Upload greenkeeper-lockfile
+          command: ~/.yarn/bin/greenkeeper-lockfile-upload
+  lint:
+    <<: *node8Defaults
+    steps:
+      - checkout
+      - restore-cache: *restore_yarn_cache
+      - run: *yarn_install
+      - save_cache: *save_yarn_cache
+      - run:
+          name: Run linters
+          command: yarn lint
+  test_unit_node_4:
+    <<: *node4Defaults
+    steps:
+      - checkout
+      - restore-cache: *restore_yarn_cache
+      - run: *yarn_install
+      - save_cache: *save_yarn_cache
+      - run: *unit_test
+  test_unit_node_6:
+    <<: *node6Defaults
+    steps:
+      - checkout
+      - restore-cache: *restore_yarn_cache
+      - run: *yarn_install
+      - save_cache: *save_yarn_cache
+      - run: *unit_test
+  test_unit_node_8:
+    <<: *node8Defaults
+    steps:
+      - checkout
+      - restore-cache: *restore_yarn_cache
+      - run: *yarn_install
+      - save_cache: *save_yarn_cache
+      - run: *unit_test_with_coverage
+  test_integration:
+    <<: *node8Defaults
+    steps:
+      - checkout
+      - restore-cache: *restore_yarn_cache
+      - run: *yarn_install
+      - save_cache: *save_yarn_cache
+      - run:
+          name: Run integration tests
+          command: yarn test:integration
+  prepare_release:
+    <<: *node8Defaults
+    steps:
+      - checkout
+      - restore-cache: *restore_yarn_cache
+      - run: *yarn_install
+      - save_cache: *save_yarn_cache
+      - run:
+          name: Run prepare release
+          command: ./.circleci/prepare-release-packages.sh
+  perform_release:
+    <<: *node8Defaults
+    steps:
+      - checkout
+      - restore-cache: *restore_yarn_cache
+      - run: *yarn_install
+      - save_cache: *save_yarn_cache
+      - run:
+          name: Run perform release
+          command: ./.circleci/perform-release-packages.sh
+
+workflows:
+  version: 2
+  build_and_test_and_maybe_release:
+    jobs:
+      - greenkeeper_lockfile
+      - lint
+      - test_unit_node_4:
+          requires:
+            - lint
+      - test_unit_node_6:
+          requires:
+            - lint
+      - test_unit_node_8:
+          requires:
+            - lint
+      - test_integration:
+          context: org-global
+          filters:
+            branches:
+              only:
+                - master
```

Comment:
This means the integration tests will only run on `Master` ?

Reply:
Yeah, we had this before https://github.com/commercetools/nodejs/blob/master/.travis/scripts.sh#L13. But I think that was only disabling them for PRs coming from forks as Travis does not expose env vars onto them. I can enable them once we migrate the env vars over.

Reply:
okay. Makes sense